tasks/check-data-keys/README.md

```diff
@@ -1,7 +1,6 @@
 # check-data-keys
 
-Tekton task to check that all required information is present in the data file in order to use the
-specified system(s). If any required keys are missing, the task will fail.
+Tekton task to check that all required information is present in the data file in order to use the specified system(s). If any required keys are missing, the task will fail. It will also validate against a JSON schema, ensuring all keys follow the correct format and are allowed.
```

Comment:
I think this can be reworded now as it isn't the existing check + the schema check now; it is just the schema check

Reply:
Reworded it.